addons/sourcemod/scripting/shop.sp

```diff
@@ -200,6 +200,8 @@ public void OnPluginStart()
 	AddCommandListener(Command_Say, "say_team");
 	
 	CreateConfigs();
+	
+	Engine_Version = GetEngineVersion();
```

Comment:
What a reason to do this?

Reply:
I think, the good way is to remove this variable, because it doesn't used in code.

Reply:
> 
> 
> I think, the good way is to remove this variable, because it doesn't used in code.

Woops, my mistake, it is used, i agree with this change.